service/history/execution/mutable_state_builder_test.go

```diff
@@ -1892,3 +1894,43 @@ func TestLog(t *testing.T) {
 	assert.NotPanics(t, func() { e.logWarn("a") })
 	assert.NotPanics(t, func() { e.logError("a") })
 }
+
+func TestMutableStateBuilder_CopyToPersistence_roundtrip(t *testing.T) {
+
+	for i := 0; i <= 100; i++ {
```

Comment:
How long does this take to run? Since we're using unix seconds as the seed and creating a new fuzz instance each time isn't it possible that a majority (or all of these) are running with the same seed?

If we used the iteration count as the seed it would be deterministic (so no chance of ever flaking) and guarantee that each run is different.

Reply:
good spotting, yeah, let me use more granular seeds